study-builder/tenants/register-user-in-pepper.js

```diff
@@ -126,6 +130,16 @@ function (user, context, callback) {
             context.idToken[pepperUserGuidClaim] = user.app_metadata.user_guid;
             return callback(null, user, context);
         } else {
+            user.user_metadata = user.user_metadata || {};
+            if (user.user_metadata.first_name) {
+                pepper_params.firstName = user.user_metadata.first_name;
+                console.log('User metadata has first name = ' + pepper_params.firstName);
+            }
+            if (user.user_metadata.last_name) {
```

Comment:
First/last name can be collected via Auth0's [custom fields](https://auth0.com/docs/libraries/custom-signup), which will store user's input into their `user_metadata`. These will then be picked up here and passed to backend.

@MocanaAtBroad, if we're implementing a custom page for `testboston` instead of using the Lock, then I think we can add two new input fields and pass these along in the request to Auth0?